### PR TITLE
javascript timers 1 | tim ahn

### PR DESCRIPTION
**Pull request by tim-ahn**
Screen Shot 2020-08-19 at 6 32 23 PM

### CODE REVIEW

File: javascript-timers-1/javascript-timers-1.code-workspace [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (thebearingedge) [inline javascript-timers-1/javascript-timers-1.code-workspace:1]: wth is this

Comment (tim-ahn) [reply javascript-timers-1/javascript-timers-1.code-workspace:1]: when i dragged the folder over, it gave me two options: to copy the folder or to add something as a workspace and then it created that file

Comment (tim-ahn) [reply javascript-timers-1/javascript-timers-1.code-workspace:1]: in VS Code*

Comment (tim-ahn) [reply javascript-timers-1/javascript-timers-1.code-workspace:1]: Screen Shot 2020-08-19 at 6 40 43 PM

Comment (tim-ahn) [reply javascript-timers-1/javascript-timers-1.code-workspace:1]: initially, i clicked the Add Folder to Workspace button and it created that file

**Pull request merged**